pkg/cmd/build/build.go

```diff
@@ -95,30 +103,45 @@ func setOutputMode(outputMode string) string {
 
 }
 
+func GetRegistryConfigFromOktetoConfig(okCtx buildCtx.OktetoContextInterface) *okteto.Config {
```

Comment:
Why do we need it to be on `pkg/cmd/build`? I don't think we should grow the `pkg/cmd/build` package. I think it'd be easier if it lives under the registry package.

Reply:
IMO `registry` pkg shouldn't know about `okteto` pkg, we only need to know the behavior exposed in `registry.configInterface`

Reply:
Then this should be on the okteto pkg and be reused by each command by implementing its own interface. WDYT?